CMakeLists.txt

```diff
@@ -259,8 +259,22 @@ list(APPEND DGL_SRC ${DGL_RPC_SRC})
 
 # Configure cuda
 if(USE_CUDA)
-  dgl_config_cuda(DGL_CUDA_SRC)
+  file(GLOB_RECURSE DGL_CUDA_SRC
+    src/array/cuda/*.cc
```

Comment:
is it a better idea to move all cuda file to a single dir?

Reply:
I think they are similar to the cc implementation files. So we also need to organize them in different folders.

Reply:
+1 to this. They are dispatched from the co-located files, so I think the current method of organizing them makes a lot of sense.